frontend/src/fixtures/zaakTypeChoices.ts

```diff
@@ -8,44 +9,50 @@ const FIXTURE_ZAAKTYPE_CHOICE: Option = {
     "http://localhost:8000/catalogi/api/v1/zaaktypen/64c98539-076e-4fbf-8fec-fa86c560fb24",
 };
 
-const FIXTURE_ZAAKTYPE_CHOICES: Option[] = [
+const FIXTURE_ZAAKTYPE_CHOICES: ZaaktypeChoice[] = [
   {
     label: "Aangifte behandelen 1",
     value:
       "http://localhost:8000/catalogi/api/v1/zaaktypen/64c98539-076e-4fbf-8fec-fa86c560fb24",
+    extra: "",
```

Comment:
Urgh, I thought that I had fixed the inconsistencies between all the choices :weary: